File: Pset6/sentimental-mario-more/mario.py
import cs50


def main():
    while True:
        height = cs50.get_int("Height: ")
        if height > 0 and height < 9:
            break
    for i in range(height):
        print_row(i + 1, (height - i) - 1)
        print("  ", end="")
        print_row(i + 1, 0)
        print()


def print_row(bricks, spaces):
    for _ in range(spaces):
        print(" ", end="")
    for _ in range(bricks):
        print("#", end="")


main()

File: CS50x/Pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int bricks, int spaces);

int main(void)
{
    int x;
    do
    {
        x = get_int("Size of the pyramid: ");
    }
    while (x < 1);

    for (int i = 0; i < x; i++)
    {
        print_row(i + 1, x - i - 1);
        printf("\n");
    }
}
void print_row(int bricks, int spaces)
{
    for (int j = 0; j < spaces; j++)
    {
        printf(" ");
    }
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
}
